demo/monolayer_growth/src/sim_param.h

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------
+//
+// Copyright (C) 2021 CERN & University of Surrey for the benefit of the
+// BioDynaMo collaboration. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// See the LICENSE file distributed with this work for details.
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// -----------------------------------------------------------------------------
+
+#ifndef SIM_PARAM_H_
+#define SIM_PARAM_H_
+
+#include "biodynamo.h"
+
+namespace bdm {
+
+// Parameters specific for this simulation
+struct SimParam : public ParamGroup {
+  BDM_PARAM_GROUP_HEADER(SimParam, 1);
+  real_t const cell_diam = 20.;
```

Comment:
Can we document **all** the parameters?
```
/// Diameter of the cell
real_t cell_diam = 20;
```

I'd remove the const specifier from the arguments and try to build the `const` into the 
`const auto* sparam = param->Get<SimParam*>()` call

Reply:
Done